typescript/commands/build.py

```diff
@@ -0,0 +1,40 @@
+import sublime_plugin
+import sublime
+import os
+from ..libs.global_vars import IS_ST2
+
+
+class TypescriptBuildCommand(sublime_plugin.WindowCommand):
+    def run(self):
+        file_name = self.window.active_view().file_name()
+        directory = os.path.dirname(file_name)
+        if "tsconfig.json" in os.listdir(directory):
+            self.window.run_command("exec", {
+                "cmd": ["tsc"],
```

Comment:
Hmmm, that's very good point. I need to understand more about how this could be done, but I agree it is indeed important to keep the versions consistent. I'll try that direction.